### DISCUSSION TITLE
Error in create pages

### DISCUSSION THREAD

**Discussion by marcogiaguaro** [TheAxeDude/ZiniGo#30] [Q&A]
i receive this error:

[C: --headless --disable-gpu --print-to-pdf=C:\Zinigo/issue/431736_9.pdf --no-margins C:\Zinigo/issue/431736_9.html]

cmd.Run() failed with exec: "C:": file does not exist

. You should retry this page.Source  https://cdn2.audiencemedia.com/0a6bd727548a5af89987f1aa4017003b32f96fd0_948/5a052597228c177cfbe812847c2581194e300f771535b0e42c621719f4585400/4ab103211f4bcf934fc4d404f5e4081e4716b7189299ac9c35a8b9f681467f9c.svg

ID:  10

[C: --headless --disable-gpu --print-to-pdf=C:\Zinigo/issue/431736_10.pdf --no-margins C:\Zinigo/issue/431736_10.html]

cmd.Run() failed with exec: "C:": file does not exist

in the "issue" dir there is nothing.
Where could be the problem?

**Comment by sunny757** [comment]
Dropping a comment here in the event anyone has gotten past this error -

**Comment by tobiasj42** [comment]
I have not found a way past it yet, but it seems like the path is being built wrong since you shouldn't have both C:\ and Zinigo/issue/ in the same path. It should either be C:\Zinigo\issue\ or C:/Zinigo/issue/ . Windows vs Linux I would assume? I have the same problem running it, but I have not tried to check the code for a fix since it is well beyond my knowledge.
